Source/ApplicationServices/LibraryCommands.cs

```diff
@@ -445,9 +445,9 @@ public static int PermanentlyDeleteBooks(this IEnumerable<LibraryBook> libraryBo
         #endregion
 
         // call this whenever books are added or removed from library
-        private static void finalizeLibrarySizeChange()
+        private static void finalizeLibrarySizeChange(LibationContext context)
```

Comment:
I'm not *opposed* to these new params, I just can't tell what we've gained.

Reply:
The context is already opened, so it saves having to reopen the context before grabbing the library. Can take upwards of a second.

Reply:
> Can take upwards of a second.

Fuuuck. Then by all means, yes: let's do this.